### PR TITLE
docs: hall of fame and update list of builders

### PR DESCRIPTION
**Pull request by laurentsimon**
closes https://github.com/slsa-framework/slsa-github-generator/issues/2341

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +91,44 @@ Specifically, these workflows do not address provenance
 [verification](https://slsa.dev/spec/v1.0/verifying-artifacts). Those requirements
 must be handled separately to meet SLSA Build level 3+.
 
+### Hall of Fame
+
+#### Generation of Provenance
+
+Below is a non exhaustive list of projects that use the builders in this repository to generate provenance:
+
+1. [flask](https://github.com/pallets/flask) ![flask stars](https://img.shields.io/github/stars/pallets/flask)
```
Comment (ianlewis) [inline README.md:100]: nit: If you add the repo name as a label and github as a logo you could combine the repo name and stars badge.

Something like this
[flask stars](https://github.com/pallets/flask)

```suggestion
1. [![flask stars](https://img.shields.io/github/stars/pallets/flask?logo=github&label=pallets/flask)](https://github.com/pallets/flask)
```

File: README.md [hunk 2]
```diff
@@ -86,6 +91,44 @@ Specifically, these workflows do not address provenance
...
+1. [pdns](https://github.com/PowerDNS/pdns) ![pdns stars](https://img.shields.io/github/stars/PowerDNS/pdns?style=social) 3.1
+1. [powertools-lambda-python](https://github.com/aws-powertools/powertools-lambda-python) ![powertools-lambda-python stars](https://img.shields.io/github/stars/aws-powertools/powertools-lambda-python?style=social) 2.2
+
+
+#### Builder Creation
+
+Several builders have been built using the "Build Your Own Builder" (BYOB):
+
+1. [nodejs builder](https://github.com/slsa-framework/slsa-github-generator/tree/main/internal/builders/nodejs#readme), by @ianlewis
+1. Coming soon! [JReleaser builder](TODO), by @aalmiray
```
[the remark this answers is not in the document]
Comment (laurentsimon) [reply README.md:128]: @aalmiray can you give us a link (it's fine if it does not exist yet)

[the remark this answers is not in the document]
Comment (laurentsimon) [reply README.md:128]: @aalmiray please let us know what's best. Main website or link to builder.

File: README.md (hunk 1 above)
Comment (laurentsimon) [reply README.md:100]: great, I'll do that!

**Comment by laurentsimon** [comment]
@ianlewis I ran the markdown utility locally and updated with the fixes it provided. The tests still fail. Any ideas?

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (aalmiray) [reply README.md:108]: Here's the link to the builder https://github.com/jreleaser/release-action/tree/java#slsa-builder

File: README.md [hunk 3]
```diff
@@ -86,6 +91,24 @@ Specifically, these workflows do not address provenance
...
.../osv-scanner?logo=github&label=google/osv-scanner)](https://github.com/google/osv-scanner) [![flux2 stars](https://img.shields.io/github/stars/fluxcd/flux2?logo=github&label=fluxcd/flux2)](https://github.com/fluxcd/flux2) [![kyverno stars](https://img.shields.io/github/stars/kyverno/kyverno?logo=github&label=kyverno/kyverno)](https://github.com/kyverno/kyverno) [![flask-sqlalchemy stars](https://img.shields.io/github/stars/pallets-eco/flask-sqlalchemy?logo=github&label=pallets-eco/flask-sqlalchemy)](https://github.com/pallets-eco/flask-sqlalchemy) [![scorecard stars](https://img.shields.io/github/stars/ossf/scorecard?logo=github&label=ossf/scorecard)](https://github.com/ossf/scorecard) [![urllib3 stars](https://img.shields.io/github/stars/urllib3/urllib3?logo=github&label=urllib3/urllib3)](https://github.com/urllib3/urllib3) [![pdns stars](https://img.shields.io/github/stars/PowerDNS/pdns?logo=github&label=PowerDNS/pdns)](https://github.com/PowerDNS/pdns) [![powertools-lambda-python stars](https://img.shields.io/github/stars/aws-powertools/powertools-lambda-python?logo=github&label=aws-powertools/powertools-lambda-python)](https://github.com/aws-powertools/powertools-lambda-python)
```
Comment (aalmiray) [inline README.md:100]: FWIW JReleaser itself uses the generic builder to generate provenance https://github.com/jreleaser/jreleaser/

**Comment by laurentsimon** [comment]
pre-submits all passing. Please LGTM

**Review by ianlewis — APPROVED** [review]

**Pull request merged**